### PR TITLE
Adds a tracker for how many hugs a player facehugger has

### PR DESCRIPTION
**Pull request by BeagleGaming1**
# About the pull request

Adds total facehugs / facehugs to next age to status panel

(Note: I was unable to test this locally due to player_stats just not existing for me)
# Explain why it's good for the game

Better than them needing to manually record it

# Changelog

:cl:
qol: lets player facehuggers see their lifetime hugs on the status panel
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/castes/Facehugger.dm [hunk 1]
```diff
@@ -225,3 +226,27 @@
...
+	var/next_facehug_goal = "no more, you are already Royal!"
+	if(total_facehugs < FACEHUG_TIER_4)
+		switch(total_facehugs)
+			if(0 to FACEHUG_TIER_1) // 0 - 4
+				next_facehug_goal = FACEHUG_TIER_1
+			if(FACEHUG_TIER_1 to FACEHUG_TIER_2) // 5 - 24
+				next_facehug_goal = FACEHUG_TIER_2
+			if(FACEHUG_TIER_2 to FACEHUG_TIER_3) // 25 - 99
+				next_facehug_goal = FACEHUG_TIER_3
+			if(FACEHUG_TIER_3 to FACEHUG_TIER_4) // 100 - 999
```
Comment (harryob) [inline code/modules/mob/living/carbon/xenomorph/castes/Facehugger.dm:246]: let the people control click to figure out what the value of the define is, don't include it as a comment - they're defines so we can update them without worrying about it being outdated in other places

**Review by harryob — CHANGES_REQUESTED** [review]

**Review by Watermelon914 — CHANGES_REQUESTED** [review]
this should really not be a verb. Include it in the statpanel if you want

**Review by harryob — APPROVED** [review]
update changelog + pr name

File: code/modules/mob/living/carbon/xenomorph/castes/Facehugger.dm [hunk 2]
```diff
@@ -41,6 +41,10 @@
 	counts_for_roundend = FALSE
 	refunds_larva_if_banished = FALSE
 	can_hivemind_speak = FALSE
+	/// The lifetime hugs from this hugger
+	var/total_facehugs = 0
```
Comment (Watermelon914) [inline code/modules/mob/living/carbon/xenomorph/castes/Facehugger.dm:45]: You don't ever set this variable anywhere.

**Review by Watermelon914 — CHANGES_REQUESTED** [review]

**Pull request merged**